requirements.txt

```diff
@@ -13,3 +13,6 @@ ethereum-serpent
 repoze.lru
 gevent-websocket==0.9.4
 cachetools>=2.0.0,<3.0.0
+Flask>=0.11.1
+flask-restful
+webargs
```

Comment:
I don't see `marshmallow` being added: is it some transient dependency, or just forgotten?